src/app/view/components/dot-device-selector/dot-device-selector.component.ts

```diff
@@ -51,4 +61,10 @@ export class DotDeviceSelectorComponent implements OnInit {
             ...devices
         ];
     }
+
+    private setDropdownWidth(devices: DotDevice[]): void {
+        const maxTextOption = _.maxBy(devices, (deviceOption: DotDevice) => deviceOption.name.length).name;
```

Comment:
All this. should be done in the util, I mean, the util should receive an array of string and give me the width in px of the longest one OR 140px if it's bigger than that, because if not we're going to have this code repeated everywhere.